packages/terra-form/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Form
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-form.svg)](https://www.npmjs.org/package/terra-form)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+Components for building forms
```

Comment:
In my opinion, we should provide links to the packages that this module contains as opposed to describing them here. Otherwise, we could risk this page getting rather large when folders like validations are later included.